src/Spice86.Core/Emulator/Devices/ExternalInput/DualPic.cs

```diff
@@ -31,6 +31,14 @@ public class DualPic : DefaultIOPortHandler {
         Initialize();
     }
 
+    private long EmulatorStartTimestamp { get; init; } = System.Diagnostics.Stopwatch.GetTimestamp();
+
+    public TimeSpan Ticks { get; private set; }
+
+    public void RefreshTicks() {
```

Comment:
Remove this and use the existing method for getting the exact time spent, as described in the README of Spice86